api/capi/src/nnstreamer-capi-util.c

```diff
@@ -499,10 +556,13 @@ ml_tensors_data_create_no_alloc (const ml_tensors_info_h info,
     return ML_ERROR_OUT_OF_MEMORY;
   }
 
-  _data->num_tensors = tensors_info->num_tensors;
-  for (i = 0; i < _data->num_tensors; i++) {
-    _data->tensors[i].size = ml_tensor_info_get_size (&tensors_info->info[i]);
-    _data->tensors[i].tensor = NULL;
+  _info = (ml_tensors_info_s *) info;
+  if (_info != NULL) {
+    _data->num_tensors = _info->num_tensors;
+    for (i = 0; i < _data->num_tensors; i++) {
+      _data->tensors[i].size = ml_tensor_info_get_size (&_info->info[i]);
+      _data->tensors[i].tensor = NULL;
+    }
```

Comment:
Why do you allow to return 0-filled data if info is NULL?

Reply:
it is for data conversion in android jni.

Reply:
This function, ...create_noalloc, is called by invoke or data create() and the metadata should be available at that time.

Another point is that this is Public API. Allowing them (app developers) to have a null data-handler appears to be the root of Tizen app bugs. If you need a null data-handler for JNI, I think you need to do it without this API (or some private/hidden API).

Reply:
ml_tensors_data_create_no_alloc() is private function for internal usage.

Reply:
Oh.. ok then.